lua/nvim-treesitter/ts_utils.lua

```diff
@@ -156,6 +156,38 @@ function M.get_root_for_position(line, col, root_lang_tree)
   return nil, nil, lang_tree
 end
 
+local function box_intersect(row, col, sRow, sCol, eRow, eCol)
```

Comment:
in general_we_use_sake_case in_this_repo

Reply:
OHHH snap!  I didn't even recognize the fact that those weren't snake_case.  Haha.  It pains me to use it and I tried to conform as much as possible.   :)  Sometimes these old eyes don't see nothing.  I'll update this pr shortly.